src/TetrisGame/GamePanel.java

```diff
@@ -0,0 +1,58 @@
+package TetrisGame;
+
+import com.sun.corba.se.impl.orbutil.graph.Graph;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseWheelEvent;
+import java.util.List;
+import java.util.Set;
+/**
+ * Created by yayixu on 9/10/17.
+ */
+public class GamePanel extends BoardedPanel{
+    public static final int UNIT = 50;
+    private static final int WIDTH = 5, HEIGHT = 10;
+    private static final String PAUSE = "PAUSE";
+
+    public static int convert(double x) {
```

Comment:
Nit: method should always be placed after attributes.

Reply:
Done.